rfcs/0069-lambda-expression.md

```diff
@@ -0,0 +1,195 @@
+---
+feature: lambda_expression
+authors:
+  - "st1page"
+start_date: "2023/08/01"
+---
+
+# Lambda expression
+
+## Background
+
+To better support user's semi-structure data, We have supported `Array` and `Struct` types compatible with PG. But in some user's requirement, We found we can not support them easily.
+
+### Induction
```

Comment:
```suggestion
### Introduction
```

Reply:
Sorry. Tiny nit only. This idea really sounds good to me. Hope we will go for it :D